src/test/java/com/epam/tc/hw2/TestExercise1HomePage.java

```diff
@@ -0,0 +1,99 @@
+package com.epam.tc.hw2;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+
+public class TestExercise1HomePage {
+
+    private WebDriver webDriver;
+    private SoftAssertions softAssertions;
+    private final List<String> expectedNavigatorBarItems = new ArrayList<>(
+        List.of("HOME", "CONTACT FORM", "SERVICE", "METALS & COLORS"));
+    private final List<String> expectedBenefitTexts = new ArrayList<>(List.of("To include good practices\n"
+        + "and ideas from successful\n"
+        + "EPAM project", "To be flexible and\n"
+        + "customizable", "To be multiplatform", "Already have good base\n"
+        + "(about 20 internal and\n"
+        + "some external projects),\n"
+        + "wish to get more…"));
+    private final List<String> expectedSidebarElements = new ArrayList<>(
+        List.of("Home", "Contact form", "Service", "Metals & Colors", "Elements packs"));
+    private final int expectedIconsNumber = 4;
+    private final String expectedUserName = "ROMAN IOVLEV";
+    private final String expectedPageTitle = "Home Page";
+
+    @BeforeClass
+    void setupWebDriver() {
+        WebDriverManager.chromedriver().setup();
+    }
+
+    @BeforeMethod
+    void createNewDriver() {
```

Comment:
It would be better if we move the driver initialization and all the setups/teardowns to the separate abstract class

Reply:
Done